README.md

```diff
@@ -11,7 +11,7 @@ Compatibility
 ------------------------------------------------------------------------------
 
 * Ember Bootstrap v4.5 or above
```

Comment:
I guess we want to drop support for Ember Bootstrap v4 in same major, aren't we?

Reply:
Idk, we *could*, but it's not absolutely required!?